webapp/channels/src/components/channel_notifications_modal/__snapshots__/channel_notifications_modal.test.tsx.snap

```diff
@@ -2713,12 +2713,12 @@ Object {
                       <header
                         class="mm-modal-header"
                       >
-                        <h1
+                        <h2
                           class="mm-modal-header__title"
                           id="mm-modal-header-channelNotificationModalLabel"
                         >
                           Notification Preferences
-                        </h1>
+                        </h2>
                         <div
```

Comment:
Given what we did in the RHS for the divider and channel name, I assume we should be consistent here too

Reply:
My only concern with this one is that the structure of the Modal header includes an extra `<div>` for the separator of the title from the subtitle, so adding the header for both would be: 

```
<h2>
    <p> Title </p>
    <div> -- divider
    <p> Subtitle </p>
</h2>
```

Any concerns with this @matthewbirtch @hmhealey ?

Reply:
I'm good with including the divider and subtitle in the `h2` tag for consistency, but we'd want to change the subtitle to be a `span` instead of a `p` then. From what I can find, it doesn't look like you're supposed to put `p` tags inside of a heading tag